src/nat/front_ends/fastapi/routes/hitl.py

```
@@ -0,0 +1,185 @@
+                if not prompts:
+                    return Response(status_code=204)
+                return HitlPendingResponse(session_id=session_id, pending=[p.model_dump() for p in prompts])
+
+            @app.post("/v1/hitl/{interaction_id}/respond")
+            async def post_response(interaction_id: str, payload: HumanResponse,
+                                    request: Request) -> HitlRespondAccepted:
+                session_id = _require_session(request)
+                try:
+                    await self._queue.respond(interaction_id, payload)
+                except KeyError:
+                    raise HTTPException(status_code=404, detail=f"Interaction {interaction_id} not found")
```

Ruff (0.14.13)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)